doc/html/hummingbird/ml/supported.html

```diff
@@ -210,8 +210,8 @@ <h1 class="title">Module <code>hummingbird.ml.supported</code></h1>
         from sklearn.decomposition import PCA, KernelPCA, FastICA, TruncatedSVD
 
         # KNeighbors models
-        from sklearn.neighbors._classification import KNeighborsClassifier
-        from sklearn.neighbors._regression import KNeighborsRegressor
+        from sklearn.neighbors import KNeighborsClassifier
```

Comment:
Why this one random html?

Reply:
ah I actually changed this because it was giving problem with sklearn 0.21.3